router_test.go

```
@@ -369,31 +373,267 @@ func Test_Router_NotFound_HTML_Inject(t *testing.T) {
 	})
 
-	resp, err := app.Test(httptest.NewRequest(MethodGet, "/dynamically-defined", nil))
+	resp, err := app.Test(httptest.NewRequest(MethodGet, "/api/feature-a", nil))
 	require.NoError(t, err, "app.Test(req)")
-	require.Equal(t, http.StatusNotFound, resp.StatusCode, "Status code")
+	require.Equal(t, http.StatusOK, resp, "Status Code")
 
-	resp, err = app.Test(httptest.NewRequest(MethodGet, "/test", nil))
-	require.NoError(t, err, "app.Test(req)")
-	require.Equal(t, http.StatusOK, resp.StatusCode, "Status code")
+	resp, err = app.Test(httptest.NewRequest(MethodGet, "/api/feature", nil))
```

golangci-lint (1.62.2)

SA4006: this value of `err` is never used

(staticcheck)